src/test/java/apprenticeship/BoardTest.java

```diff
@@ -16,44 +14,72 @@ public void initBoard() {
         tictactoe = new Board();
     }
 
-    @Test
-    public void aBoardInitializeWithNineCells() throws Exception {
-        int emptyCells = tictactoe.countCells();
-        assertEquals(emptyCells, 9);
-    }
 
     @Test
     public void canInsertSymbolOnBoard() throws Exception {
-        tictactoe.insertSymbol('x', 0);
-        char atZero = tictactoe.getSymbol(0);
-        assertEquals(atZero, 'x');
+        tictactoe.insertSymbol("X", 0);
+        String atZero = tictactoe.getSymbol(0);
```

Comment:
Just out of curiosity, why did you change `char` to `String`? Is your board able to support any arbitrary string with a length greater than one?